platform/platform_unix_impl.cpp

```diff
@@ -124,28 +124,14 @@ Platform::TStorageStatus Platform::GetWritableStorageStatus(uint64_t neededSize)
     return STORAGE_DISCONNECTED;
   }
 
-  /// @todo May be add additional storage space.
-  if (st.f_bsize * st.f_bavail < neededSize)
+  auto available = st.f_bsize * st.f_bavail;
```

Comment:
```suggestion
  auto const availableBytes = st.f_bsize * st.f_bavail;
```
The reviewer sees that the var is not edited later.

Reply:
fixed